### PR TITLE
[TW-351] Intro to SSO refresh

### PR DESCRIPTION
**Pull request by MatthewPinsker-postman**
* Removed "What is SSO?" H2, and confirmed that no other pages link to this section
* Added "Contents" section
* Added link to the new Custom SAML page, which is part of a separate PR (TW-956)
* Reorganized supported IdPs to match the navigation menu
* Ran Vale and fixed suggestions

### CODE REVIEW

File: src/pages/docs/administration/sso/intro-sso.md [hunk 1]
```diff
@@ -17,26 +14,31 @@ contextual_links:
 
 > **[SSO is available on Postman Professional and Enterprise plans.](https://www.postman.com/pricing)**
 
-## What is SSO?
-
 Single sign-on (SSO) services enable you to manage your team’s identity across all the SaaS products that you use.
 
-SSO services permit a user to use one set of credentials (for example, a name or email address and password) to access multiple applications. The service authenticates the user only once for all the applications the user has been given rights to and eliminates further prompts when the user switches applications during the same session.
+SSO services let a user to use one set of credentials (for example, a name or email address and password) to access multiple applications. The service authenticates the user only once for all the applications the user has been given rights to and eliminates further prompts when the user switches applications during the same session.
```
Comment (mark-anthony-postman) [inline src/pages/docs/administration/sso/intro-sso.md:19]:
```suggestion
With an SSO service, a user can access multiple applications using one set of credentials (for example, an email address and password). The SSO service authenticates the user once for all the applications the user has been given rights to and eliminates further prompts when the user switches applications during the same session.
```

File: src/pages/docs/administration/sso/intro-sso.md [hunk 2]
```diff
@@ -17,26 +14,31 @@ contextual_links:
...
+* [SSO setup for SAML 2.0 compliant IdPs](#sso-setup-for-saml-20-compliant-idps)
 
-### What are the advantages of SSO?
+## What are the advantages of SSO?
 
 * Removes the need for users to remember and manage multiple passwords.
-* Simplifies users' experience by allowing them to sign in at one single access point and enjoy a seamless experience across multiple applications.
+* Simplifies users' experience by allowing them to sign in at a single access point and enjoy a seamless experience across multiple applications.
 * Increases productivity by significantly reducing the password-related support emails.
 * Reduces phishing and thereby making sure users aren't tricked into giving away sensitive information.
```
Comment (mark-anthony-postman) [inline src/pages/docs/administration/sso/intro-sso.md:35]:
```suggestion
* Reduces phishing attempts that try to trick users into giving away sensitive information.
```

File: src/pages/docs/administration/sso/intro-sso.md [hunk 3]
```diff
@@ -17,26 +14,31 @@ contextual_links:
...
+* [Prerequisites for SSO with Postman](#prerequisites-for-sso-with-postman)
+* [Supported IdPs](#supported-idps)
+* [SSO setup for SAML 2.0 compliant IdPs](#sso-setup-for-saml-20-compliant-idps)
 
-### What are the advantages of SSO?
+## What are the advantages of SSO?
 
 * Removes the need for users to remember and manage multiple passwords.
-* Simplifies users' experience by allowing them to sign in at one single access point and enjoy a seamless experience across multiple applications.
+* Simplifies users' experience by allowing them to sign in at a single access point and enjoy a seamless experience across multiple applications.
```
Comment (mark-anthony-postman) [inline src/pages/docs/administration/sso/intro-sso.md:33]:
```suggestion
* Simplifies users' experience by enabling them to sign in at a single access point and work seamlessly across multiple applications.
```

File: src/pages/docs/administration/sso/intro-sso.md (hunk 1 above)
Comment (mark-anthony-postman) [reply src/pages/docs/administration/sso/intro-sso.md:19]: "SSO services let a user to use..." needs rewriting 😄

File: src/pages/docs/administration/sso/intro-sso.md [hunk 4]
```diff
@@ -46,21 +48,21 @@ Professional plans:
 
 Enterprise plans:
 
+* [Microsoft AD FS](/docs/administration/sso/microsoft-adfs/)
+* [Azure AD](/docs/administration/sso/azure-ad/)
+* [Custom SAML](/docs/administration/sso/custom-saml/)
```
Comment (mark-anthony-postman) [inline src/pages/docs/administration/sso/intro-sso.md:53]: I assume you'll merge your other PR first so the Custom SAML link here will work?

**Review by mark-anthony-postman — APPROVED** [review]
Great page refresh! Made a few wording suggestions you can use if you like 😀

File: src/pages/docs/administration/sso/intro-sso.md (hunk 4 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/administration/sso/intro-sso.md:53]: My plan is to merge them at the same time, and then test on the `develop` branch. What do you think?

Comment (MatthewPinsker-postman) [reply src/pages/docs/administration/sso/intro-sso.md:53]: Yes, I agree it makes sense to first merge the PR with the Custom SAML page, and then pull the changes into this branch for testing. Thanks!

File: src/pages/docs/administration/sso/intro-sso.md [hunk 5]
```diff
@@ -46,21 +48,21 @@ Professional plans:
...
-* [Custom SAML](/docs/administration/sso/admin-sso/)
 
-### SSO setup for SAML 2.0 compliant IdPs
+## SSO setup for SAML 2.0 compliant IdPs
 
-Most SAML 2.0 compliant identity providers require the same information about the service provider for setup (Postman is the service provider). These values are specific to a Postman team and are available after configuring SSO in the Edit Team Page.
+Most SAML 2.0 compliant identity providers require the same information about the service provider for setup (Postman is the service provider). These values are specific to a Postman team and are available after [configuring SSO](/docs/administration/sso/admin-sso/).
 
-Learn more about [setting up SSO](/docs/administration/sso/admin-sso/).
+> While configuring your IdP, make sure the user’s email address is set in attribute statements.
```
Comment (PriyambadaMadala) [inline src/pages/docs/administration/sso/intro-sso.md:64]: Can we highlight this part. This is essential for the SSO set up .

Comment (PriyambadaMadala) [reply src/pages/docs/administration/sso/intro-sso.md:64]: The rest of the changes looks good to me .

**Review by ArjunSingh-PM — APPROVED** [review]
The changes look good.

**Review by PriyambadaMadala — APPROVED** [review]

File: src/pages/docs/administration/sso/intro-sso.md (hunk 5 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/administration/sso/intro-sso.md:64]: Thanks for bringing this up. I created ticket `TW-1053` to look into this in more detail.

File: src/pages/docs/administration/sso/intro-sso.md [hunk 6]
```diff
@@ -46,21 +48,21 @@ Professional plans:
...
 
-**Note** - While configuring your IdP, make sure the user’s email address is set in attribute statements.
+<!-- -->
 
-**Note** - Postman’s form-based authentication mechanism and Google OAuth 2.0 remain enabled even if SSO is configured for a team.
+> Postman’s form-based authentication mechanism and Google OAuth 2.0 remain enabled even if SSO is configured for a team.
```
Comment (ArjunSingh-PM) [inline src/pages/docs/administration/sso/intro-sso.md:68]: These auth methods are not automatically disabled when SSO is configured but the team admin can disable these if so desired.

**Review by ArjunSingh-PM — APPROVED** [review]
The changes look good. Thanks!

File: src/pages/docs/administration/sso/intro-sso.md (hunk 6 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/administration/sso/intro-sso.md]: This is now updated. Please see my latest commits. Thanks!

**Pull request merged**